test/functionalTest/cases/117_convop_using_standard_ops/getContextEntitiesByEntityId.test

```diff
@@ -0,0 +1,646 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+ConvOp getContextEntitiesByEntityId: /v1/registry/contextEntities/{entityId::id}
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# This test case verifies that the convop 'getContextEntitiesByEntityId' does everything
```

Comment:
Yeah, I thought I would have some 'clear' use case with URI params.
But, as all the URI param treatment is inside mongo backend, not inside the service routine postDiscoverXXX, I don't know exactly what to add to the func-test.
I did add service-path, but if you can think of some more test case, that would be great,

We want to make sure that this conv op does *everything* that the standard op does.
That is what the test is for ...